sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/AzureOpenAIOperationState.java

```diff
@@ -26,13 +26,6 @@ public final class AzureOpenAIOperationState extends ExpandableStringEnum<AzureO
     /** The operation has completed processing with a failure and cannot be further consumed. */
     @Generated public static final AzureOpenAIOperationState FAILED = fromString("failed");
 
-    /**
```

Comment:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/AzureOpenAIOperationState.java#L34

Reply:
I think no? https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/images/common.tsp#L9-L29